engine/menus/save.asm

```diff
@@ -37,6 +37,7 @@ LoadSAV0:
 	ld a, $1
 	ld [MBC1SRamBankingMode], a
 	ld [MBC1SRamBank], a
+	vc_hook SaveLimit ; This hook is ambigiously located here. Can be Moved.
 	ld hl, sPlayerName ; hero name located in SRAM
```

Comment:
Let's introduce `sGameData` and `sGameDataEnd` labels to replace the usage of `sPlayerName` and `sMainDataCheckSum - sPlayerName` here and in the template.